### PR TITLE
Integrate hubot

### LINKED ISSUE #5: Integrar com o hubot

### PR DESCRIPTION
**Pull request by askmon**
Resolves #5 

Transform devices bot into a hubot script

### CODE REVIEW

**Comment by askmon** [comment]
@dandpb @tibawatanabe

File: src/awesome.coffee [hunk 1]
```diff
@@ -35,11 +35,12 @@ module.exports = (async, config) ->
     request path, (error, response, body) ->
       console.log devicesEndpoint + '?' + params
       if (!error && response.statusCode == 200 && body)
+        console.log body
```
Comment (felipesabino) [inline src/awesome.coffee]: `console.log`

File: src/awesome.coffee [hunk 2]
```diff
@@ -168,22 +169,24 @@ module.exports = (async, config) ->
         callback null, "Device was removed"
 
   executeCommand = (text, user, callback) ->
+    console.log text
```
Comment (felipesabino) [inline src/awesome.coffee:144]: `console.log`

**Comment by tibawatanabe** [comment]
👍

**Comment by askmon** [comment]
Hey @felipesabino , @tibawatanabe I've updated this PR. I've replaced json server with hubot integrated reddis, so I think we have an MVP. 

Tested hubot integration already.

File: src/awesome.coffee [hunk 3]
```diff
@@ -1,237 +1,211 @@
...
+        args.splice(0, 1);
+        helpText = "Yo " + user.name + "! How r u doin, bro? Im here to help you find and manage devices. For example, an iPhone 6 Plus iOS 9.2 64 GB White. \n\nYou can type \`want-device\` followed by any term of what you want to find (e.g., \*want-device iPhone\* or \*want-device iOS\*). \n\nOr, you can just ask for the availability of all devices by typing \`want-device all\`. When you take a device, \nremember to tell others by typing \`got-device\` followed by its id. When you return a device, \ntell your bros too: type \`back-device\` followed by its id.\n\nYou can also register a new device or delete an existing one by typing \`register-device\` or \`delete-device\`, followed by its full information. Remember, bro, You will need: \*id\*, \*model\*, \*os\*, \*version\*, \*notes\*, \*owner\* (e.g., register-device \"Blackberry 1\" \"Blackberry Curve\" \"blackberry\" \"7.0.0\" \"Bundle 2055 black\" \"your name\")."
```
Comment (felipesabino) [inline src/awesome.coffee]: @askmon 👍  por colocar `device-` no início, mas agora o help ficou desatualizado 😢

**Comment by askmon** [comment]
Fixed help text

**Comment by felipesabino** [comment]
👍

**Pull request merged**